egs/imagenet/v1/image/bmp_encoder.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
I needed to change the /image/nnet3/get_egs.sh file so it can't be a symlink to ../../cifar/v1/image